src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -9061,9 +9061,9 @@ void VRFTerminalUnitEquipment::ControlVRFToLoad(EnergyPlusData &state,
         }
     } else if (VRFCoolingMode || HRCoolingMode) {
         // IF the system is in cooling mode and/or the terminal unit requests cooling
-        if (NoCompOutput <= QZnReq && ((QZnReq <= 0.0) || (QZnReq >= DataHVACGlobals::SmallLoad && !HRCoolingMode))) {
+        if (NoCompOutput <= QZnReq) {
             DXCoolingCoilOprCtrl = false;
-            if (!this->SuppHeatingCoilPresent) {
+            if (!this->SuppHeatingCoilPresent || HRCoolingMode) {
                 PartLoadRatio = 0.0;
                 return;
             }
```

Comment:
In the original issue the code here was working correctly. What happend was the code fell through here and returned instead of activating dropping through and activating the supplemental heating coil. So a new `bool DXCoolingCoilOprCtrl` was added as well as a test to allow VRFTU's with supp heating coils to drop through. The problem was, when this code dropped through, the cooling and heating coil was not sure what to do.